pages/geral/index.js

```diff
@@ -1,18 +1,93 @@
+
+    for (var i = 0; i < variables.length; i++) {
+        var currVariable = variables[i];
+        ret[currVariable] = $("body").attr("tmp_" + currVariable);
+        $("body").removeAttr("tmp_" + currVariable);
+    }
+
+    $("#tmpScript").remove();
+
+    return ret;
+}
+
 const initGcBooster = async () => {
-    if ($('#GamersClubStatsBox').is(':visible')) {
-        const minPontos = $('.StatsBoxProgressBar__minRating').text();
-        const maxPontos = $('.StatsBoxProgressBar__maxRating').text();
-        const atualPontos = $('.StatsBoxRating__Score').text();
-        const pontosSubir = maxPontos - atualPontos;
-        const pontosCair = minPontos - atualPontos;
-        $('.StatsBoxRating__Header').append(`<span style="font-size:10px">${pontosCair} / ${pontosSubir}</span>`);
+    if ( generalOptions.autoMostrarLevelProgress ) {
+        var windowVariables = retrieveWindowVariables(["PLAYERID"]);
+        const PlayerID = windowVariables.PLAYERID;      //$(`#GamersClubStatsBox`).attr( "data-prop-player-id" ); old method
+        $.get( "https://gamersclub.com.br/api/box/init/" + parseInt(PlayerID) ).done( function( data ) {
+            const playerName = data.playerInfo.nick;
+            const playerLevel = data.playerInfo.level;
+            const currentRating = data.playerInfo.rating;
+
+            const minPontos = XpRangeFromLevel(playerLevel).minRating;
+            const maxPontos = XpRangeFromLevel(playerLevel).maxRating;
+
+            const pontosCair = minPontos - currentRating;
+            const pontosSubir = maxPontos - currentRating;
+
+            const playerNextLevel = playerLevel == 20 ? 21 : playerLevel + 1;
+            const progressBar = maxPontos ? `${((currentRating - minPontos) / (maxPontos - minPontos)) * 100}%` : '100%';
+
+            const strText = playerNextLevel == 21 ? "" : "Skill Level " + playerNextLevel;
+            const nextLvl = playerNextLevel == 21 ? "" : playerNextLevel;
+
+            var fixedNum = parseFloat(progressBar).toFixed(4);
+
+            $('.MainHeader__navbarBlock:last').before(`<div style="display: flex;align-items: center;margin-right: 24px;margin-left: 24px;">
+                <span title="Skill Level ${playerLevel}" style="display: inline-block;" data-tip-text="Skill Level ${playerLevel}">
+                <div class="PlayerLevel PlayerLevel--${playerLevel} PlayerLevel--nonSubscriber" style="height: 28px; width: 28px; font-size: 12px;"><div class="PlayerLevel__background"><span class="PlayerLevel__text">${playerLevel}</span></div></div>
+                </span>
+                <div style="margin-right: 4px;margin-left: 4px;">
+                    <div class="text-light" style="display: flex; justify-content: space-between;">
+                        <a class="text-sm text-muted bold" style="align-self: flex-end;">
+                            <div style="overflow: hidden;text-overflow: ellipsis;width: 70%;">${playerName}</div>
+                        </a>
```

Comment:
```suggestion
                        <div class="text-sm text-muted bold" style="align-self: flex-end;"></div>
```

Removeria o nome, já que já possui na própria GC...